app/src/common/urls.js

```diff
@@ -181,6 +181,8 @@ export const URLS = {
   suite: (activeProject, suiteId) => `${urlBase}${activeProject}/item/${suiteId}`,
 
   notification: (activeProject) => `${urlBase}notification/${activeProject}`,
+  deleteNotification: (activeProject, notificationId) =>
+    `${urlBase}notification/${activeProject}/${notificationId}`,
```

Comment:
You don't use any 'delete' word in the url itself, so it must be self-explained, f.e.:
`notificationById`, as this endpoint can also be used to update notification.

Reply:
Updating passes without ID,